section2/reflection.md

```diff
@@ -1,13 +1,22 @@
 ## Section 2 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+* I definitely have a tendency to wait too long. I think it is more out of stubbornness than embarrassment. I would always rather figure it out myself. The blog bit that stood out most for me was the author talking about how they spent a long time googling so as not to inconvenience their boss, but when they did give up and ask, the boss was able to answer in mere moments. It's good to remember that everyone's time is valuable, but that includes your own. If someone else has the information readily available, sometimes it's better to use them as a resource rather than struggle through it yourself.
```

Comment:
This is excellent to reflect on. I often try to remind myself when I'm too embarrassed to ask a question that someone else may have the same question and by having the courage to ask it helps them as well.